pkg/inputs/mongobatch/input.go

```diff
@@ -27,6 +27,9 @@ type Config struct {
 	BatchSize      int                     `mapstructure:"batch-size"  toml:"batch-size" json:"batch-size"`
 	WorkerCnt      int                     `mapstructure:"worker-cnt" toml:"worker-cnt" json:"worker-cnt"`
 	ChunkThreshold int                     `mapstructure:"chunk-threshold"  toml:"chunk-threshold"  json:"chunk-threshold"`
+	// IgnoreOplogError ignores error with oplog.
+	// Some mongo cluster setup may not support oplog.
+	IgnoreOplogError bool `mapstructure: "ignore-oplog-error" toml:"ignore-oplog-error" json:"ignore-oplog-error"`
```

Comment:
actually I think the better approach(for all kinds of input) is handling the position login in two stage input rather than in batch input, add get current position interface into specific stream input.